posthog/templates/email/invite.html

```diff
@@ -0,0 +1,19 @@
+{% extends "email/base.html" %} {% load posthog_assets %} {% load posthog_filters %}
+
+{% block main %}
+<div class="container">
+    <div class="section">
+        <h1>Join {{ invite.organization.name }} on PostHog</h1>
+        <p>
+            <strong>{{ invite.created_by.first_name }}</strong> ({{ invite.created_by.email }}) invited you to join organization {{ invite.organization.name }} on PostHog.<br/>
+            PostHog is an open-source product analytics tool that will help you better understand your users.
+        </p>
+        <p><a class="button" href="{{ invite }}">Join {{ invite.organization.name }} Now</a></p>
+        <p>
+            If the button above does not work, paste the following link into your browser:<br/>
+            <a href="{{ invite }}">{{ invite }}</a>
+        </p>
+        <p>The invite will expire in 3 days.<br/>If you received this by mistake, please delete it.</p>
+    </div>
+</div>
+{% endblock %}
```

Comment:
For this particular email, the footer does need to be overridden, because you wouldn't be able to change your subscription preferences for this. You would need a message along the lines of: You received this message because X invited you to join Y on PostHog. If you are receiving these messages by mistake, please reply to this email.